File: 6/Six.java
/**
 * Solves Project Euler problem #6
 * https://projecteuler.net/problem=6
 * 
 * @author Jake Conway
 * @version 1.0
 */
public class Six
{
    public static void main(String[] args)
    {
        int totalSq = 0;
        int totalAd = 0;
        for(int i = 1; i <=100; i++)
        {
            totalSq += Math.pow(i, 2);
            totalAd += i;
        }
        totalAd = (int) Math.pow(totalAd, 2);
        System.out.println("Solution: " + (totalAd-totalSq));
    }
}

File: Solutions/1-9/6/six.py
class six:
  def main():
    totalSq = 0
    totalAd = 0
    for x in range(1, 101):
      totalSq += (x**2)
      totalAd += x
      
    totalAd = totalAd**2
    print("Solution: " + str(totalAd - totalSq))
if __name__ == '__main__':
  six.main()
